### ISSUE TITLE
adding snippets

### ISSUE THREAD

**Issue by jess** [carlhuda/janus#375]
Can anyone recommend a way to add snippets without editing the core snippets dir or snipmate plugin?  

What I've tried so far that _hasn't worked_:
### Add a snippets dir to `~/.janus`

Just added my snippets I'm used to using in the janus dir hoping it would pick it up

**IMO, this would be the perfect solution :-)**
### Disable snipmate-snippets and add snipmate-snippets to `~/.janus`

```
# .vimrc.before
call janus#disable_plugin('snipmate-snippets')"

git clone https://github.com/honza/snipmate-snippets.git ~/.janus/
```

The disabling of snipmate-snippets actually works, even though it's not actually a plugin, but adding it back did nothing.
### Disabling all snipmate dependencies and the snipmate plugin and trying to add a custom version back

Basically the same as above, but I included all the plugins/dependencies listed here https://github.com/garbas/vim-snipmate.  I followed the pathogen install instructions, swapping bundle with .janus

I know I must be doing something wrong, I  wouldn't think that everyone is using the default snippets??
